### ISSUE TITLE
Autopilot option for LND

### ISSUE THREAD

**Issue by Jacksper13** [mynodebtc/mynode#514]
I am missing the autopilot feature that LND offers, could be helpful to keep the manual maintenance to a minimum.

Ideally, there would be an autopilot switch that would lead to a dropdown menu that would allow to edit some of the below options. Adding these uncommented to lnd.conf does the trick (have manually tested it).

I would say important things to edit are autopilot maxchannels, allocation, min and maxsize, default private to false but maybe offer it as option, and the block target maybe?

Copied and pasted from [here](https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf):

[autopilot]

; If the autopilot agent should be active or not. The autopilot agent will
; attempt to automatically open up channels to put your node in an advantageous
; position within the network graph.
; autopilot.active=true

; The maximum number of channels that should be created.
; autopilot.maxchannels=5

; The fraction of total funds that should be committed to automatic channel
; establishment. For example 0.6 means that 60% of the total funds available
; within the wallet should be used to automatically establish channels. The total
; amount of attempted channels will still respect the maxchannels param.
; autopilot.allocation=0.6

; Heuristic to activate, and the weight to give it during scoring. (default:
; top_centrality:1)
; autopilot.heuristic=preferential:1

; The smallest channel that the autopilot agent should create (default: 20000)
; autopilot.minchansize=20000

; The largest channel that the autopilot agent should create (default: 16777215)
; autopilot.maxchansize=20000

; Whether the channels created by the autopilot agent should be private or not.
; Private channels won't be announced to the network.
; autopilot.private=true

; The minimum number of confirmations each of your inputs in funding transactions
; created by the autopilot agent must have. (default: 1)
; autopilot.minconfs=2

; The confirmation target (in blocks) for channels opened by autopilot. (default:
; 3)
; autopilot.conftarget=2

- [x] Mark this if you searched the existing list of open/closed issue and did not find anything similar

**Comment by Jacksper13** [comment]
@tehelsper is this in the roadmap? It's a pain to add them manually to the config file every timr there's an update and then have a "custom" config file... Thanks!

**Comment by tehelsper** [comment]
Yeah, that's got to be a pain. I'll think about it, but I'm not sure how many people use autopilot. Maybe I'll post a poll on Twitter or something to see if it can be prioritized.
